.travis.yml

```diff
@@ -138,12 +96,8 @@ before_script:
   - mysql -e "GRANT ALL PRIVILEGES ON *.* TO 'travis'@'localhost' WITH GRANT OPTION;";
   - psql -U postgres -c 'CREATE DATABASE yiitest;';
 
-  # enable code coverage on PHP 7.1, only one PHP version needs to generate coverage data
-  - |
-    if [ $TRAVIS_PHP_VERSION = '7.1' ]; then
-      PHPUNIT_FLAGS="--coverage-clover=coverage.clover"
-    fi
```

Comment:
this should be kept as it now applies to 7.1 and nightly and as soon as 7.2 is out we need it again.